test/basic.test.js

```diff
@@ -159,10 +159,17 @@ describe("basic", function() {
 		css.should.match(/\.other/);
 		css.should.match(/a\.button/);
 	});
-  it("imports the right file based on context", function() {
-    var css = require("!raw-loader!..!./fixtures/context");
-    (typeof css).should.be.eql("string");
-    css.should.match(/\.a-color/);
-    css.should.match(/\.b-color/);
-  });
+	it("imports the right file based on context", function() {
+		var css = require("!raw-loader!..!./fixtures/context");
+		(typeof css).should.be.eql("string");
+		css.should.match(/\.a-color/);
+		css.should.match(/\.b-color/);
+	});
+	it("imports the right file based on relative path", function() {
+		var css = require("!raw-loader!..!./fixtures/relative");
+		(typeof css).should.be.eql("string");
+		css.should.match(/\.a-color/);
+		css.should.match(/\.b-color/);
+		css.should.match(/\.c-color/);
+	});
```

Comment:
Looks like an existing test was indented with spaces, while the convention was already tabs. Fixed the existing test.